Let's pimp the readme file a bit

Screenfeed opened the pull request:
Added 2 badges to the readme file: the "version" from Packagist and the license from Github.
Also fixed the "version" in the "Installation" section.

Screenfeed committed:
Let's pimp the readme file a bit
Added 2 badges to the readme file: the "version" from Packagist and the license from Github.
Also fixed the "version" in the "Installation" section.

Chouby commented:
I propose to:
- Remove the mention to `(PHP 5.6 and WP 5.4)` as I just removed in #6.
- Keep the version `^1.0` in the installation instruction.
- Merge this PR and then tag the v1.0.

Screenfeed committed:
Removed mention of specific PHP 5.6 and WP 5.4 versions
But added the related tags in the example file.

Screenfeed commented:
@Chouby I removed the specific mention of `(PHP 5.6 and WP 5.4)` and, while at it, I added the 2 related config tags to the example.

Screenfeed committed:
Use a static badge for the packagist version
Since we don't create releases, the dynamic badge displays unexpected results.
Also, moved from `badgen.net` to `shields.io`.

Screenfeed commented:
> Maybe it's not useful to tag? What do you think?

I don't think it is right now.
It may be useful if someday we make impacting changes, like forbidding Yoda conditions: in that case we could tag 1.0 before the changes and use 1.0 in projects that are not ready yet.
